src/finput.js

```diff
@@ -197,6 +202,11 @@ class Finput {
     this.setValue(newValue, false);
   }
 
+  isModifierKeyPressed(originalEvent) {
+    const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
+    return isMac ? originalEvent.metaKey : originalEvent.ctrlKey;
```

Comment:
Unfortunately we do need to check for mac platform as `metaKey` is the windows key (on windows) or command key (on mac). Windows uses ctrl as modifier, but mac uses command as modifier - https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/metaKey. 

`originalEvent` is the event raised by the browser so doesn't have the `modifierKey` key. In fact we use this function to add `modifierKey` to our own "event":
```
const keyEvent = {
  keyName: e.key.toLowerCase(),
  modifierKey: this.isModifierKeyPressed(e)
};
```

Perhaps I can clear all this up by:
- adding documentation to the function as per rest of the file
- renaming our `keyEvent` to something else to distinguish from the original browser event

Let me know if you think those changes are worth adding.